ghost/core/core/server/services/members/members-api/repositories/MemberRepository.js

```diff
@@ -948,7 +948,6 @@ module.exports = class MemberRepository {
             // Use first Ghost product as default product in case of missing link
             if (!ghostProduct) {
                 ghostProduct = await this._productRepository.getDefaultProduct({
-                    forUpdate: true,
                     ...options
```

Comment:
Object destructuring is not required now. 
Can be : `await this._productRepository.getDefaultProduct(options);`